docs/topics/modules/glossary.adoc

```diff
@@ -120,6 +141,12 @@ iteration:: A development cycle used to organize, plan, and execute work items i
 // endterm
 ////
 
+////
+// term: 49d1a19f-02b4-4a10-a774-5723299f8944, en
+(agile) iteration:: A development cycle used to organize, plan, and execute work items in a certain order. It comprises a logical mix of work items slated to be executed within the time frame of the iteration.
```

Comment:
_...work items ... to be executed ..._? I don't think WIs are executed. Rather, they're worked on, completed, finished, etc.